tests/foreman/ui/test_packages.py

```diff
@@ -22,21 +42,63 @@ def test_positive_search_in_repo(self):
         @Assert: Content search functionality works as intended and expected
         packages are present inside of repository
         """
-        org = entities.Organization().create()
-        product = entities.Product(
-            name=gen_string('alpha'),
-            organization=org,
-        ).create()
-        yum_repo = entities.Repository(
-            name=gen_string('alpha'),
-            product=product,
-            content_type='yum',
-            url=FAKE_0_YUM_REPO,
-        ).create()
-        yum_repo.sync()
-
         with Session(self.browser) as session:
-            session.nav.go_to_select_org(org.name)
-            self.package.select_repo(yum_repo.name)
+            session.nav.go_to_select_org(self.organization.name)
+            self.package.select_repo(self.yum_repo.name)
             self.assertIsNotNone(self.package.search('bear'))
             self.assertIsNotNone(self.package.search('cheetah'))
+
+    @tier2
+    def test_positive_search_in_multiple_repos(self):
+        """Create product with two different yum repositories assigned to it.
+        Search for packages inside of these repositories. Make sure that unique
+        packages present in corresponding repos.
+
+        @Feature: Packages
+
+        @Assert: Content search functionality works as intended and expected
+        packages are present inside of repositories
+        """
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.organization.name)
+            self.assertIsNotNone(self.package.search('tiger'))
+            self.assertIsNotNone(self.package.search('Lizard'))
+            self.package.select_repo(self.yum_repo.name)
+            self.assertIsNotNone(self.package.search('tiger'))
+            self.assertIsNone(self.package.search('Lizard'))
+            self.package.select_repo(self.yum_repo2.name)
+            self.assertIsNotNone(self.package.search('Lizard'))
+            self.assertIsNone(self.package.search('tiger'))
+
+    @tier2
+    def test_positive_check_package_details(self):
+        """Create product with yum repository assigned to it. Search for
+        package inside of it and then open it. Check all the details about that
+        package
+
+        @Feature: Packages
+
+        @Assert: Package is present inside of repository and has all expected
+        values in details section
+        """
+        with Session(self.browser) as session:
+            session.nav.go_to_select_org(self.organization.name)
+            self.package.select_repo(self.yum_repo.name)
+            self.package.check_package_details(
+                'gorilla',
+                [
+                    ['Description', 'A dummy package of gorilla'],
```

Comment:
Hm... On the one hand - i don't like this test as all the data is hardcoded inside the test without any variables/constants so it's not flexible in case something happens with that repo, on the other - too many values for constant to declare (imo) and basically single-use constant is not ok too...
Is there any good way to fetch repo information via api/cli? If yes, we could fetch that data and verify ui shows the same values.